support-frontend/assets/helpers/subscriptionsForms/submit.js

```diff
@@ -80,42 +76,40 @@ function getAddresses(state: AnyCheckoutState) {
   };
 }
 
-const getOptions = (
-  fulfilmentOptions: FulfilmentOptions,
-  productOptions: ProductOptions,
-): {fulfilmentOptions: FulfilmentOptions, productOptions: ProductOptions} =>
-  ({
-    ...(fulfilmentOptions !== NoFulfilmentOptions ? { fulfilmentOptions } : {}),
-    ...(productOptions !== NoProductOptions ? { productOptions } : {}),
```

Comment:
I wasn't quite sure about the whole "No...Options" being omitted, as on the server side they are required in all cases, so if they were omitted there would either be a deserialisation error or the reader would get transparently subscribed to the wrong product (e.g. Guardian Weekly instead of newspaper)